docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+To set up an uptime monitor, you will pull an Elastic-provided Docker image, configure the monitor,
+and use the Docker image to run the monitor and send data back to Elasticsearch.
+After setup is complete, the data will be available in the {uptime-app} in {kib} to view, analyze, and alert on.
```

Comment:
While I did advise that we should at least mention running heartbeat outside of docker here as a possibility. It's still popular, but maybe we can say that we recommend docker because it reduces the chance of problems.